src/components/employees/EmployeeDetails.jsx

```diff
@@ -1,3 +1,85 @@
+import { useEffect, useState } from "react"
+import { useNavigate, useParams } from "react-router-dom"
+
 export const EmployeeDetails = () => {
-  return <>EmployeeDetails</>;
-};
+  // get employeeId from the url
+  const { employeeId } = useParams()
+  // state to hold fetch
+  const [employee, setEmployee] = useState(null)
+  // error if fetch fails
+  const [error, setError] = useState(null)
+
+  useEffect(() => {
+    const getEmployeeDetails = async () => {
+      try {
+        const response = await fetch(
+          `http://localhost:8088/employees/${employeeId}`
+        )
+        if (!response.ok) {
+          throw new Error("Failed to get employee details")
+        }
+        const details = await response.json()
+        setEmployee(details)
+      } catch (error) {
+        setError(error.message)
+      }
+    }
+
+    getEmployeeDetails()
+  }, [employeeId]) //dependency array to trigger when employeeId changes
+
+  // handle delete employee
```

Comment:
There is a service for this in employeeServices that you can call inside your handler function.

Reply:
then handleEditEmployee will use call the `updateEmployee()` function passing in the employee object.